WordPress/UITestsFoundation/Screens/Login/LoginEpilogueScreen.swift

```diff
@@ -41,12 +34,12 @@ public class LoginEpilogueScreen: BaseScreen {
     public func verifyEpilogueDisplays(username: String? = nil, siteUrl: String) -> LoginEpilogueScreen {
         if var expectedUsername = username {
             expectedUsername = "@\(expectedUsername)"
-            let actualUsername = usernameField.label
+            let actualUsername = app.staticTexts["login-epilogue-username-label"].label
```

Comment:
@jostnes [raised a valid question](https://github.com/wordpress-mobile/WordPress-iOS/pull/17599/files#r763740311) on whether to adopt this inline style vs moving everything at the start of the file.

I stuck to the approach I used so far because: 1) we haven't taken a decision yet; 2) it's what (most of) the rest of the `ScreenObject` subclasses here use.